web/packages/teleterm/src/ui/components/CatchError/CatchError.jsx

```diff
@@ -39,7 +39,7 @@ export default class CatchError extends React.Component {
   render() {
     if (this.state.error) {
       return (
-        <Failed alignSelf={'baseline'} message={this.state.error.message} />
```

Comment:
`FailedApp` is equivalent to `Failed`, but it also adds `StyledApp` and `StaticThemeProvider`.

Reply:
Hmm, actually that's bad because in case of an error we render `StyledApp` twice (one time in `App` and one time in `CatchError`). I'm just going to move `CatchError` to the top of the hierarchy in `App` then.